afpacket/afpacket.go

```diff
@@ -27,6 +27,8 @@ import (
 	"golang.org/x/sys/unix"
 
 	"github.com/google/gopacket"
+	"github.com/google/gopacket/layers"
+	"github.com/google/gopacket/pcap"
```

Comment:
The reason we don't have this yet is that I'd like afpacket to compile if folks don't have libpcap installed... this creates a dependency on libpcap for afpacket-based binaries, which I don't think is ideal.